vis/visualization/activation_maximization.py

```diff
@@ -12,50 +12,69 @@
 
 def visualize_activation_with_losses(input_tensor, losses, wrt_tensor=None,
                                      seed_input=None, input_range=(0, 255),
-                                     **optimizer_params):
+                                     input_indices=0, **optimizer_params):
```

Comment:
we should just return outputs for each input_tensors

Reply:
I think, the result is different from when Optimizer was supported multiple `wrt_tensors`.

Reply:
Is it no problem?